src/ui/widget.cpp

```diff
@@ -1592,8 +1592,8 @@ bool Widget::onProcessMessage(Message* msg)
       // Propagate the message to the parent.
       if (parent())
         return parent()->sendMessage(msg);
-      else
-        break;
+
+      break;
```

Comment:
Is this change necessary? I say this because it looks like a style/format unrelated change and when I merge this to beta I'll just ignore the change (as beta already contains another fix).

Reply:
No, not at all. Just a small improvement, but if there are changes already, I will take it back right now.